vscode-extension/syntaxes/rascal.tmLanguage.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Changes in this file are due to the "extra" change in this PR

Reply:
ah, it's going to be a blast of a merge conflict with my other small PR. which might be a reason to pull those changes out of this PR and apply them on main after merging that?

Reply:
Merged `main` into this branch, then resolved all conflicts in favor of `main`, then regenerated the file